CppCoreGuidelines.md

```diff
@@ -2376,7 +2376,7 @@ Other function rules:
 * [F.54: If you capture `this`, capture all variables explicitly (no default capture)](#Rf-this-capture)
 * [F.55: Don't use `va_arg` arguments](#F-varargs)
 
-Functions have strong similarities to lambdas and function objects.
+Functions have a lot of similarities to lambdas and function objects.
```

Comment:
fixed as suggested

Reply:
Strong isn't quantitative, it's qualitative.
Changing this from `strong` to `many` or `a lot of` removes the existing "behavior" of describing the `similarities`. Essentially this change as is (and as is updated) just expands upon the plurality of the `similarities`.

Reply:
Reverted. @Lirusaito is correct of course, though I could argue that since the document does not further enumerate or qualify each similarity, it apparently does not matter much how many or how strong those similarities are, or which ones they are. The sentence as a whole does not seem to contain much valuable content, nor does this position in the text seem to contribute much, probably the sentence could be deleted without reducing the value of the styleguide. However, for this PR, only inclusive language matters, and actually this usage of stengths does not equate stronger==better, so I just reverted.